tower_cli/models/base.py

```diff
@@ -1204,18 +1205,14 @@ class Resource(ResourceMethods):
                   help='If used, if a match is found on unique fields, other '
                        'fields will be updated to the provided values. If '
                        'False, a match causes the request to be a no-op.')
-    def create(self, fail_on_found=False, force_on_exists=False, **kwargs):
+    def create(self, **kwargs):
         """Create an object.
 
         Fields in the resource's `identity` tuple are used for a lookup;
         if a match is found, then no-op (unless `force_on_exists` is set) but
         do not fail (unless `fail_on_found` is set).
         """
-        return super(Resource, self).write(
-            create_on_missing=True,
-            fail_on_found=fail_on_found, force_on_exists=force_on_exists,
-            **kwargs
-        )
+        return self.write(create_on_missing=True, **kwargs)
```

Comment:
This is just consolidating code here. Even if you give explicit default, click will still force `fail_on_found=None` if the user didn't provide a value, so we're just repeating ourselves.